spinoso-time/src/time/chrono/build.rs

```diff
@@ -73,4 +106,25 @@ mod tests {
         let time = Time::default();
         assert_eq!(time.offset, Offset::Local);
     }
+
+    #[test]
+    fn time_at_with_seconds_and_sub_second_nanos() {
+        let time = Time::at(100, 100);
+        assert_eq!(time.timestamp, 100);
+        assert_eq!(time.sub_second_nanos, 100);
+    }
+
+    #[test]
+    fn time_at_with_overflowing_sub_second_nanos() {
+        let time = Time::at(100, 1_000_000_001);
+        assert_eq!(time.timestamp, 101);
+        assert_eq!(time.sub_second_nanos, 1);
+    }
+
+    #[test]
+    fn time_at_with_negative_sub_second_nanos() {
+        let time = Time::at(100, -1);
+        assert_eq!(time.timestamp, 99);
+        assert_eq!(time.sub_second_nanos, 999_999_999);
+    }
```

Comment:
These tests are great!

Can we add one more that exercises the "time too large flow" once we add the checked addition?